### DISCUSSION TITLE
邮箱发送错误

### DISCUSSION THREAD

**Discussion by Wangji5673** [bs-community/blessing-skin-server#533] [Q&A]
错误报告如下
Connection could not be established with host :stream_socket_client(): php_network_getaddresses: getaddrinfo failed: Name or service not known

配置如下
MAIL_MAILER=smtp
MAIL_HOST=smtp.qq.com
MAIL_PORT=465
MAIL_USERNAME=XXXXXXXXXX@qq.com
MAIL_PASSWORD=XXXXXXXXXXXX
MAIL_ENCRYPTION=ssl
MAIL_FROM_ADDRESS=
MAIL_FROM_NAME=

**Comment by CharCaius** [comment]
请在本地尝试使用nslookup smtp.qq.com检查DNS解析是否正常
